sql/core/benchmarks/InExpressionBenchmark-results.txt

```diff
@@ -2,550 +2,739 @@
-InSet expression                               268 /  269         37.3          26.8       0.5X
+In expression                                  244 /  262         41.0          24.4       1.0X
+InSet expression                               225 /  238         44.5          22.5       1.1X
 
-OpenJDK 64-Bit Server VM 1.8.0_191-b12 on Linux 3.10.0-862.3.2.el7.x86_64
-Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_192-b12 on Mac OS X 10.14.3
+Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
 25 dates:                                Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------
-In expression                                  196 /  196         51.1          19.6       1.0X
-InSet expression                               277 /  282         36.1          27.7       0.7X
+In expression                                  274 /  282         36.5          27.4       1.0X
+InSet expression                               241 /  268         41.4          24.1       1.1X
 
-OpenJDK 64-Bit Server VM 1.8.0_191-b12 on Linux 3.10.0-862.3.2.el7.x86_64
-Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_192-b12 on Mac OS X 10.14.3
+Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
 50 dates:                                Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------
-In expression                                  338 /  351         29.5          33.8       1.0X
-InSet expression                               287 /  290         34.9          28.7       1.2X
+In expression                                  304 /  353         32.9          30.4       1.0X
+InSet expression                               230 /  253         43.4          23.0       1.3X
 
-OpenJDK 64-Bit Server VM 1.8.0_191-b12 on Linux 3.10.0-862.3.2.el7.x86_64
-Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_192-b12 on Mac OS X 10.14.3
+Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
 100 dates:                               Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------
-In expression                                  541 /  542         18.5          54.1       1.0X
-InSet expression                               299 /  300         33.5          29.9       1.8X
+In expression                                  404 /  472         24.8          40.4       1.0X
+InSet expression                               240 /  266         41.7          24.0       1.7X
 
-OpenJDK 64-Bit Server VM 1.8.0_191-b12 on Linux 3.10.0-862.3.2.el7.x86_64
-Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_192-b12 on Mac OS X 10.14.3
+Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
 200 dates:             
```

Comment:
Ur, this PR is irrelevant to this ratio change, isn't it?

Reply:
No, it has no effect on this. I assume we see such a difference because of machines. [My original evaluation](https://github.com/apache/spark/pull/23291/commits/4c9d7b625c899fbe72c8fc4cad94fa4195630ab8) had a similar ratio as we see now.

Also, I re-tested this PR on a t2.xlarge EC2 instance. 
```
[info] OpenJDK 64-Bit Server VM 1.8.0_191-b12 on Linux 4.14.77-70.59.amzn1.x86_64
[info] Intel(R) Xeon(R) CPU E5-2686 v4 @ 2.30GHz
[info] 200 structs:                             Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
[info] ------------------------------------------------------------------------------------------------
[info] In expression                                 2614 / 2895          0.4        2614.5       1.0X
[info] InSet expression                               427 /  433          2.3         427.3       6.1X
```